app/demo-geant-integration/HitClassesLinkDef.h

```diff
@@ -0,0 +1,10 @@
+#ifdef __ROOTCLING__
```

Comment:
Needs a header like `celeritas/ext/RootInterfaceLinkDef.h`. Also why does that file use `#ifdef __CINT__` instead of `__ROOTCLING__`? Is the former a deprecated (or otherwise out of date) macro?

Reply:
> Also why does that file use #ifdef __CINT__ 

Because the author forgot that Cint was retired about 9 years ago and that defined is still there only as a courtesy but should not be used for new files (and I missed it during PR review I guess ;) )

Reply:
humm ... I am not sure there is a concise way to answer that phrasing :) .... Anyway, I will add it :)